.gitlab-ci.yml

```diff
@@ -205,12 +206,14 @@ deploy:review:
   when: manual
   except:
   - master
+  allow_failure: false
 
 deploy:master:
   <<: *deploy_template
   when: always
   only:
   - master
+  allow_failure: false
```

Comment:
This is default, instead of it better to set to true only when needed

Why at all we allow to fail deploy?

Reply:
It was to prevent allowing it later. It's removed now